data/tlg5034/tlg001a/tlg5034.tlg001a.perseus-grc1.xml

```diff
@@ -48,20 +48,23 @@
                         <monogr>
                             <author>Scholia Pindar</author>
                             <title>Scholia in Pindarum Olympian Odes</title>
+                            <editor>A. B. Drachmann</editor>
                             <imprint>
                                 <pubPlace>Leipzig</pubPlace>
-                                <publisher>B.G. Teubneri</publisher>
+                                <publisher xml:lang="lat">B.G. Teubneri</publisher>
                                 <date>1903</date>
                             </imprint>
-                            <biblScope unit="volume">1</biblScope>
+                            
                         </monogr>
+                        <series><title level="s" xml:lang="lat">Scholia in Olympionicas</title><biblScope n="volume">Vol. 1</biblScope></series>
```

Comment:
Remove the `Vol. ` from the text of the `<biblScope>` element here. Since "volume" is already specified in the `n` attribute, we don't need it in the text.